src/index.css

```
@@ -356,23 +356,37 @@ html {
   background:
     radial-gradient(120% 55% at 50% -20%, rgba(43, 140, 238, 0.18), transparent 60%),
     linear-gradient(180deg, #f8fbff 0%, #eef4ff 100%);
+  height: 100dvh;
+  overflow: hidden;
 }
 
 .onboarding-body {
   font-family: "Manrope", "DM Sans", "Segoe UI", Tahoma, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)